src/screens/DateInput.js

```diff
@@ -147,6 +148,13 @@ const DateInputPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="readOnlyCopy">
+          <Description>
+            Makes the TextInput readOnly and adds a copy to clipboard button.
```

Comment:
were we going to mention that they can use the `readOnly` alone or since that's a native behavior were not mentioning it? I don't think its own section but maybe within this description but not strong opinion if we want to leave off

Reply:
Yeah I didn't document it because it is a native prop. But maybe I could add something in the `readOnlyCopy` description

Reply:
Just pushed up changes. I just mentioned the `readOnly` prop more explicitly within the `readOnlyCopy` description so hopefully that helps